clientv3/balancer.go

```diff
@@ -128,13 +137,17 @@ func (b *simpleBalancer) Notify() <-chan []grpc.Address { return b.notifyCh }
 
 func (b *simpleBalancer) Close() error {
 	b.mu.Lock()
+	defer b.mu.Unlock()
+	if b.closed {
```

Comment:
grpc can call `Close` more than once too? if not, should probably omit the check?